dspace-api/src/main/java/org/dspace/statistics/SolrLoggerUsageEventListener.java

```diff
@@ -29,15 +29,18 @@
 
 	private static Logger log = Logger.getLogger(SolrLoggerUsageEventListener.class);
 
-    @Autowired(required = true)
     protected SolrLoggerService solrLoggerService;
 
+	public SolrLoggerUsageEventListener() {
+		solrLoggerService = new DSpace().getServiceManager().getServiceByName("solrLoggerService", SolrLoggerService.class);
```

Comment:
Could you retrieve this solr logger service with the following factory call: StatisticsServiceFactory.getInstance().getSolrLoggerService();